### ISSUE TITLE
We have no way to show all issue trees

### ISSUE THREAD

**Issue by matthiasbeyer** [neithernut/git-dit#146]
Right now we lack a way to `git dit show -t` all issues, am I right?

Or even list all issues with `-g` (one might want that...)

**Comment by neithernut** [comment]
I don't think that a variant of `git dit show` showing all issues makes much sense. Neither does a graph-option for `git dit list`.

However, it _does_ make sense to offer a switch for including referenced issues in `show` or `list`. In the case of a list subcommand, we would list the (filtered) issues and display a tree of referenced issues for each entry.

**Comment by matthiasbeyer** [comment]
Hm, my initial message was badly worded... I don't think we need a `show` command showing all issues, ... but _some_ command showing all issues like `show -t`/`show -g` does. Whether this is the `list` command or the `show` command is irrelevant to me.
